common/services/prismic/opening-times.ts

```diff
@@ -355,14 +355,13 @@ export function parseCollectionVenues(
 export function getTodaysVenueHours(
   venue: Venue
 ): ExceptionalOpeningHoursDay | OpeningHoursDay | undefined {
-  const todaysDate = london().startOf('day');
+  const todaysDate = london();
```

Comment:
is `.startOf('day')` superfluous?

Reply:
No, I just [moved it ](https://github.com/wellcomecollection/wellcomecollection.org/blob/214734da23f9f287657cda72a613d7ed0fa8dd7c/common/services/prismic/opening-times.ts#L363)so it was clearer what was being compared in the find method